app/views/users/registrations/new.html.erb

```diff
@@ -9,29 +9,21 @@
   <%= hidden_field_tag :invitation_token, resource.invitation_token %>
 
   <fieldset>
-    <legend><%= ts("User Details") %></legend>
+    <legend><%= t(".legend.user") %></legend>
     <%= render :partial => "passwd", :locals => {:f => f} %>
   </fieldset>
 
   <fieldset>
-    <legend><%= ts("Legal Agreements") %></legend>
+    <legend><%= t(".legend.legal") %></legend>
     <%= render :partial => "legal", :locals => {:f => f} %>
   </fieldset>
 
   <fieldset>
-    <legend><%= ts("Submit") %></legend>
+    <legend><%= t(".submit") %></legend>
     <p class="submit cancel actions">
-      <input type="submit" name="cancel_create_account" value="Cancel" />
-      <%= submit_tag ts('Create Account') %>
+      <%= link_to t(".cancel"), root_path %>
+      <%= submit_tag t(".submit") %>
     </p>
   </fieldset>
 <% end %>
 <!--/content-->
-
-<%= content_for :footer_js do %>
-  <%= javascript_tag do %>
-    $j(document).ready(function(){
-        $j('#user_registration_form').preventDoubleSubmit();
-     })
-  <% end %>
```

Comment:
I'm not sure I like the idea of completely removing the code preventing double submissions, since the Rails code for rescuing `RecordNotUnique` errors and showing the confirmation text anyway has been removed. Would it work to prevent duplicate submissions in a way similar to the work form? https://github.com/otwcode/otwarchive/blob/702394ddc76edd81c901f81710c135db1365657f/app/views/works/_standard_form.html.erb#L353-L354

Reply:
Can do.

Reply:
I see — thanks for explaining. I will stick to using "Please wait..." then.